### PR TITLE
Fix WSL paths

### LINKED ISSUE #2217: Incorrect WSL path on windows mounted drive
I created a simple test notebook on my H: drive in windows but using WSL.  The notebook is shown below.  After exiting and trying to reload the notebook from WSL again, I get the following error:

## REPL command
```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.7.3 (2022-05-06)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using Pluto
run
julia> Pluto.run(notebook="test_notebook.jl")
[ Info: Loading...
    Updating registry at `~/.julia/registries/General.toml`
┌ Info:
└ Opening http://localhost:1236/open?secret=6fCu0YAo&path=test_notebook.jl in your default browser... ~ have fun!
┌ Info:
│ Press Ctrl+C in this terminal to stop Pluto
└
```

## Error in browser

```
Failed to load notebook
The file /mnt/h/test_notebook.jl could not be loaded. Please [report this error](https://github.com/fonsp/Pluto.jl/issues)!


[Go back](http://localhost:1236/open?secret=6fCu0YAo&path=test_notebook.jl#)


Error message:

IOError: realpath("/mnt/h/mnt/h/test_notebook.jl"): no such file or directory (ENOENT)
```

I noticed in the error that the `/mnt/h` is repeated in the path which seems to be why it cannot find it.  I tried other julia commands to open the file and tested the `realpath` function and julia within wsl does seem to know the correct path for the file in general.

```julia
julia> realpath("test_notebook.jl")
"/mnt/h/test_notebook.jl"
```

## Notebook file `test_notebook.jl`
```julia
### A Pluto.jl notebook ###
# v0.19.9

using Markdown
using InteractiveUtils

# ╔═╡ 44fcc996-070c-11ed-0818-b717a531c573
1+1

# ╔═╡ 00000000-0000-0000-0000-000000000001
PLUTO_PROJECT_TOML_CONTENTS = """
[deps]
"""

# ╔═╡ 00000000-0000-0000-0000-000000000002
PLUTO_MANIFEST_TOML_CONTENTS = """
# This file is machine-generated - editing it directly is not advised

julia_version = "1.7.3"
manifest_format = "2.0"

[deps]
"""

# ╔═╡ Cell order:
# ╠═44fcc996-070c-11ed-0818-b717a531c573
# ╟─00000000-0000-0000-0000-000000000001
# ╟─00000000-0000-0000-0000-000000000002
```

### PR DESCRIPTION
**Pull request by pabloferz**
Closes #2217

### CODE REVIEW

File: src/notebook/path helpers.jl [hunk 1]
```diff
@@ -28,7 +28,8 @@ function maybe_convert_path_to_wsl(path)
     try
         if detectwsl()
             # wslpath utility prints path to stderr if it fails to convert
-            # (it fails for WSL-valid paths)
+            # (it used to fail for WSL-valid paths)
+			startswith(path, "/mnt/c/") && return path
```
Comment (pabloferz) [inline src/notebook/path helpers.jl:32]:
```suggestion
            startswith(path, "/mnt/c/") && return path
```

**Review by rikhuijzer — COMMENTED** [review]
Thanks for taking a look, @pabloferz! Do you run WSL and have you verified the patch on your system?

**Comment by pabloferz** [comment]
Yes. I have verified that it runs fine locally.

**Comment by jekyllstein** [comment]
I did try using the PR but had the same error.  However, if I first used `cd ~` in wsl before launching julia and pluto, then I could open the notebook by pasting the windows path into the text box.

File: src/notebook/path helpers.jl [hunk 2]
```diff
@@ -28,7 +28,8 @@ function maybe_convert_path_to_wsl(path)
     try
         if detectwsl()
             # wslpath utility prints path to stderr if it fails to convert
-            # (it fails for WSL-valid paths)
+            # (it used to fail for WSL-valid paths)
+            startswith(path, "/mnt/c/") && return path
```
Comment (pankgeorg) [inline src/notebook/path helpers.jl:32]:
```suggestion
             isnothing(match(r"^/mnt/\w+/", path)) && return path
```

Comment (jekyllstein) [reply src/notebook/path helpers.jl]: Just tested the new change and this did work on my system.  Thanks!

**Pull request merged**